src/docs/getting-started.mdx

```diff
@@ -63,3 +63,11 @@ import { Button } from "@involves/violin";
 
 <Button>Enjoy!</Button>
 ```
+
+or you can import individualy components. This way you don't load all library to using only some component.
```

Comment:
Trocar `or you can import individualy components.`
Por `or you can import individual components.`